scripts/install_third_party.sh

```diff
@@ -134,7 +136,7 @@ echo Checking if pylint is installed in $TOOLS_DIR
 if [ ! -d "$TOOLS_DIR/pylint-1.9.3" ]; then
   echo Installing Pylint
 
-  pip install pylint==1.9.3 --target="$TOOLS_DIR/pylint-1.9.3"
+  pip install pylint==1.9.3 --system --target="$TOOLS_DIR/pylint-1.9.3"
```

Comment:
FYI, I think whether this works or not depends on the user's system. I took a look into it a bit, and from what I can tell, it depends on what the system-wide --user configuration is set up as. 

Not sure how to make this failsafe -- maybe refactor `pip install` into a bash function that tries one and then the other? I suspect the change here will break some installations.

Reply:
Great point--done.